CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - ds-rshiny cleanup cloudera dependency ([#540](https://github.com/opendevstack/ods-quickstarters/pull/540))
 - ds-rshiny quickstarter goes to broken repository ([#605](https://github.com/opendevstack/ods-quickstarters/issues/605))
+- be-gateway-nginx switch from CentOS to Fedora ([#611](https://github.com/opendevstack/ods-quickstarters/issues/611))
```

Comment:
switch is from alpine, right? Is the alpine image not maintained anymore?

Reply:
Good catch! just backported all to 3.x, in 3.x we were still using alpine and was fully unmaintained, we migrated to centos in the past and that was not backported to 3.x. The reason to move to centos in the past (and now fedora) was mainly due to better OS dependencies support.